schemas/input/csl-citation.json

```diff
@@ -40,11 +40,9 @@
               "$ref": "#/definitions/locators"
             }
           },
-          "suppress-author": {
-            "type": ["string", "number", "boolean"]
-          },
-          "author-only": {
-            "type": ["string", "number", "boolean"]
+          "command": {
+            "type": "string",
+            "enum": ["author-in-text", "suppress-author", "author-only"]
```

Comment:
```suggestion
            "type": "string",
            "enum": ["author-in-text", "suppress-author", "author-only"]
```
```suggestion
            "type": "string",
            "title": "Special citation command",
            "descriptions": "Calls for a non-default citation format."
            "enum": ["narrative", "suppress-author", "author-only"]
```

Reply:
I'll wait on feedback from some of the others on this; it's one of the questions I added to the main post.

Thanks.